recipes/libspnav/meta.yaml

```diff
@@ -0,0 +1,48 @@
+{% set name = "libspnav" %}
+{% set version = "0.2.3" %}
+{% set sha256 = "7ae4d7bb7f6a5dda28b487891e01accc856311440f582299760dace6ee5f1f93" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}.tar.gz
+  url: https://github.com/FreeSpacenav/{{ name }}/releases/download/{{ name }}-{{ version }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  skip: true  # [win]
+
+requirements:
+  build:
+    - {{ compiler("c") }}  # [unix]
```

Comment:
Since this is Unix-only (and will likely stay this way), we can remove the `  # [unix]` selectors since they are always `  # [True]`.